src/main/java/com/psychology/product/util/ResponseHandler.java

```diff
@@ -0,0 +1,25 @@
+package com.psychology.product.util;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import java.util.Map;
+
+public class ResponseHandler {
+    public static ResponseEntity<Object> generateResponse(String messege, HttpStatus status) {
+        Map<String, Object> map = Map.of(
+                "message", messege,
+                "status", status.value()
+        );
+        return new ResponseEntity<>(map, status);
+    }
+
+    public static ResponseEntity<Object> generateError(String message, HttpStatus status) {
+        Map<String, Object> map = Map.of(
+                "message", message,
+                "status", status.value()
+        );
+        return new ResponseEntity<>(map, status);
```

Comment:
Why we return twice status, in response and in the body?

Reply:
We get status.value then return status